queries.sql

```diff
@@ -79,4 +79,56 @@ WHERE escape_attempts = 0 AND owners.full_name = 'Dean Winchester';
 SELECT owners.full_name, COUNT(animals.species_id) as count FROM owners
 LEFT JOIN animals ON owners.id = animals.owner_id
 GROUP BY owners.id ORDER BY count DESC
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
+SELECT animals.name, vet.name, visits.visit_date FROM animals
+INNER JOIN visits on animals.id = visits.animal_id
+INNER JOIN vet on visits.vet_id = vet.id
+WHERE vet.name = 'William Tatcher'
+ORDER BY visits.visit_date DESC lIMIT 1;
+
+SELECT vet.name, count(animals.name) FROM animals
+INNER JOIN visits on visits.animal_id = animals.id
+INNER JOIN vet on vet.id = visits.vet_id
+WHERE vet.name = 'Stephanie Mendez'
+GROUP BY vet.name;
+
+SELECT vet.name, species.name FROM vet
+LEFT JOIN specializations on specializations.vet_id = vet.id
+LEFT JOIN species on specializations.species_id = species.id;
+
+SELECT animals.name, vet.name, visits.visit_date FROM animals
+INNER JOIN visits on visits.animal_id = animals.id
+INNER JOIN vet on vet.id = visits.vet_id
+WHERE vet.name = 'Stephanie Mendez' AND visits.visit_date
+BETWEEN '2020-04-01' AND '2020-08-30';
+
+SELECT animals.name, count(*) FROM animals
+INNER JOIN visits on visits.animal_id = animals.id
+GROUP BY animals.name
+ORDER BY count desc limit 1;
+
+SELECT animals.name, vet.name, visits.visit_date FROM animals
+INNER JOIN visits on animals.id = visits.animal_id
+INNER JOIN vet on visits.vet_id = vet.id
+WHERE vet.name = 'Maisy Smith'
+ORDER BY visits.visit_date ASC lIMIT 1;
+
+SELECT animals.name, vet.name, visits.visit_date FROM animals
+INNER JOIN visits on animals.id = visits.animal_id
+INNER JOIN vet on visits.vet_id = vet.id
+ORDER BY visits.visit_date DESC lIMIT 1;
+
+SELECT COUNT(*) FROM vet
+INNER JOIN visits ON visits.vet_id = vet.id
+INNER JOIN animals ON visits.animal_id = animals.id
+INNER JOIN specializations ON vet.id = specializations.vet_id
+WHERE NOT specializations.species_id = animals.species_id;
```

Comment:
- [ ] THis query should return the number of visits that were with a vet that didn't specialize in that animals species. Keeping in mind that Maisy smith does not have a specialty, so all her visits should be counted, Jack harness specialized on digimon so any visit that is not a digimon animal should be counted and William tatcher specialized on pokemon so any visit that is not pokemon should've counted, the query should return 12visits but from your screenshots, 7visits were returned. Kindly fix this query please and add the right screenshot.
![image](https://user-images.githubusercontent.com/94551484/236539821-be639b82-a331-4e8b-8184-b874d0127b32.png)